controllers/operandrequest/reconcile_operator.go

```diff
@@ -218,6 +223,27 @@ func (r *Reconciler) deleteSubscription(ctx context.Context, operandName string,
 		return nil
 	}
 
+	// check and remove registry in annotation of subscription
+	regName := registryInstance.ObjectMeta.Name
+	regNs := registryInstance.ObjectMeta.Namespace
+	delete(sub.Annotations, regNs+"."+regName)
+	if len(sub.Annotations) != 0 {
+		// remove the registry from subscription and remain the resource present
```

Comment:
maybe change comment to be more clear

```suggestion
		// remove the associated registry from annotation of subscription
```